MapleServer2/Types/Guild.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using MaplePacketLib2.Tools;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using MapleServer2.Tools;
+
+namespace MapleServer2.Types
+{
+    public class Guild
+    {
```

Comment:
do we want to add guild properties later (like guild map, level, rank names, etc?) Maybe drop a todo